### PR TITLE
Add instructions to readme for certificate rollover

### PR DESCRIPTION
**Pull request by mattkim**
Add instructions to readme for certificate rollover

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,22 @@ tcld namespace update accepted-client-ca set -n <namespace> --ca-certificate-fil
 ```
 > :warning: If the update removes a certificate, any clients (tctl/workers) still using the removed certificate will fail to connect to the namespace after the update completes.
 
+#### Performing a Certificate Rollover:
```
Comment (djmagee) [inline README.md:56]:
```suggestion
#### Performing a certificate rollover:
```

File: README.md [hunk 2]
```diff
@@ -53,6 +53,22 @@ tcld namespace update accepted-client-ca set -n <namespace> --ca-certificate-fil
...
+1. Create a single file that contains both your old and new CA cert pem blocks. You can do this by simply concatenating each pem block on a new line.
+```
+-----BEGIN CERTIFICATE-----
+... old CA cert ...
+-----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
+... new CA cert ...
+-----END CERTIFICATE-----
+```
+2. Run the update accepted-client-ca command with the new CA cert bundle file.
```
Comment (djmagee) [inline README.md:67]:
```suggestion
2. Run the `update accepted-client-ca` command with the new CA cert bundle file.
```

File: README.md [hunk 3]
```diff
@@ -53,6 +53,22 @@ tcld namespace update accepted-client-ca set -n <namespace> --ca-certificate-fil
 ```
 > :warning: If the update removes a certificate, any clients (tctl/workers) still using the removed certificate will fail to connect to the namespace after the update completes.
 
+#### Performing a Certificate Rollover:
+It's important to do a rollover process when updating your CA certificates. This allows your namespace to serve both CA certs for a period of time until traffic to your old cert is gone. To do this follow these steps:
```
Comment (djmagee) [inline README.md:57]: In the interest of global English, I recommend spelling out "cert" globally (`s/cert/certificate/`). If not, shorten "certificate" in step 3 for consistency.

File: README.md [hunk 4]
```diff
@@ -53,6 +53,22 @@ tcld namespace update accepted-client-ca set -n <namespace> --ca-certificate-fil
...
+... old CA cert ...
+-----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
+... new CA cert ...
+-----END CERTIFICATE-----
+```
+2. Run the update accepted-client-ca command with the new CA cert bundle file.
+3. Monitor traffic to your old certificate until it is gone.
+4. Create a new CA cert bundle file with the old cert removed.
+5. Run the update accepted-client-ca command again with the new file.
```
Comment (djmagee) [inline README.md:70]:
```suggestion
5. Run the `update accepted-client-ca` command again with the new file.
```

**Review by djmagee — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -53,6 +53,22 @@ tcld namespace update accepted-client-ca set -n <namespace> --ca-certificate-fil
 ```
 > :warning: If the update removes a certificate, any clients (tctl/workers) still using the removed certificate will fail to connect to the namespace after the update completes.
 
+#### Performing a Certificate Rollover:
+It's important to do a rollover process when updating your CA certificates. This allows your namespace to serve both CA certs for a period of time until traffic to your old cert is gone. To do this follow these steps:
+1. Create a single file that contains both your old and new CA cert pem blocks. You can do this by simply concatenating each pem block on a new line.
```
Comment (djmagee) [inline README.md:58]:
```suggestion
1. Create a single file that contains both your old and new CA cert PEM blocks. You can do this by simply concatenating the PEM blocks on adjacent lines.
```

**Review by djmagee — APPROVED** [review]
Added one more comment as I integrated this content into the tcld reference.

File: README.md (hunk 5 above)
Comment (mattkim) [reply README.md:58]: small nit here: I feel that programmers might understand "new line" more than "adjacent lines"

**Review by djmagee — APPROVED** [review]

**Pull request merged**